### PR TITLE
423

### PR DESCRIPTION
**Pull request by sisskind**
#423

### CODE REVIEW

**Comment by sisskind** [comment]
@brianhatchl Way to test if you'd like is to load review dataset and delete all red/blue features.  Should get "no review" warning, but software should save changes and not melt down.

File: js/hoot/control/conflicts/actions/IdGraphSynch.js [hunk 1]
```diff
@@ -47,8 +47,14 @@ Hoot.control.conflicts.actions.idgraphsynch = function (context)
                     callback(f);
                 }
 
-            }
+            } 
         } else {
+            // Check to see if the relation has been deleted.  If so, do not continue...
+            if((_.find(context.history().changes().deleted,{id:fid}))){
+                _validateMemberCnt(fid,null);
```
Comment (mv003348) [inline js/hoot/control/conflicts/actions/IdGraphSynch.js]: Codacy Issue found: ["_validateMemberCnt" was used before it was defined](https://www.codacy.com/app/DGIS/hootenanny-ui/file/2565108061/issues/source?bid=3294725&fileBranchId=3294729#l54)

**Comment by sisskind** [comment]
Believe the "melting" issue of repetitive error messages is mitigated, but seems like save vs. resolved is not acting in same manner.  Resolved yields no error messages, while saved appears to try to load the features back in or does not recognize that they have been deleted.  Will continue to pull this thread.

**Comment by sisskind** [comment]
Error returned: 
> d3.min.js:1 GET http://localhost:8080/hoot-services/osm/api/0.6/relations?mapId=74&elementIds=2 404 (Not Found)

**Comment by brianhatchl** [comment]
Is id 2 the deleted review relation?

**Comment by sisskind** [comment]
@brianhatchl believe so

**Comment by sisskind** [comment]
Closing this pull request in favor of new one to avoid merging issues with reverting code.

**Pull request closed without merging**